<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeTest</class>
 <widget class="QDialog" name="HomeTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Home_HomeTest_lbSampleCode">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeTest_lbSampleInfo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeTest_lbSampleRemark">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeTest_lbTestName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeTest_lbTestSerial">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeTest_lbTestType">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Home_HomeTest_btCancelTest">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeTest_lbBK1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeTest_lbBK2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Home_HomeTest_lbBK2</zorder>
  <zorder>Home_HomeTest_lbBK1</zorder>
  <zorder>Home_HomeTest_lbSampleCode</zorder>
  <zorder>Home_HomeTest_lbSampleInfo</zorder>
  <zorder>Home_HomeTest_lbSampleRemark</zorder>
  <zorder>Home_HomeTest_lbTestName</zorder>
  <zorder>Home_HomeTest_lbTestSerial</zorder>
  <zorder>Home_HomeTest_lbTestType</zorder>
  <zorder>Home_HomeTest_btCancelTest</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
